diff -rup alsa-lib-1.0.27.2-orig/src/userfile.c alsa-lib-1.0.27.2/src/userfile.c
--- alsa-lib-1.0.27.2-orig/src/userfile.c	2014-05-15 22:33:24.439939337 +0300
+++ alsa-lib-1.0.27.2/src/userfile.c	2014-05-15 23:00:23.422990322 +0300
@@ -32,7 +32,7 @@
  * stores the first matchine one.  The returned string is strdup'ed.
  */
 
-#ifdef HAVE_WORDEXP_H
+#if 0
 #include <wordexp.h>
 #include <assert.h>
 int snd_user_file(const char *file, char **result)
