diff -rup alsa-utils-1.0.28-orig/configure.ac alsa-utils-1.0.28/configure.ac
--- alsa-utils-1.0.28-orig/configure.ac	2014-08-01 22:14:26.028770937 +0300
+++ alsa-utils-1.0.28/configure.ac	2014-08-01 22:16:02.280769213 +0300
@@ -287,7 +287,7 @@ case "$dir" in
 *) dir="$prefix/share"
 esac
 
-soundsdir="$dir/sounds/alsa"
+soundsdir="$dir/alsa/sounds"
 AC_DEFINE_UNQUOTED(SOUNDSDIR, "$soundsdir", [directory containing sample data])
 
 mydatadir="$dir/alsa"
diff -rup alsa-utils-1.0.28-orig/speaker-test/samples/Makefile.am alsa-utils-1.0.28/speaker-test/samples/Makefile.am
--- alsa-utils-1.0.28-orig/speaker-test/samples/Makefile.am	2014-08-01 22:14:26.022770937 +0300
+++ alsa-utils-1.0.28/speaker-test/samples/Makefile.am	2014-08-01 22:15:40.184769609 +0300
@@ -1,4 +1,4 @@
-sounddir = $(datadir)/sounds/alsa
+sounddir = $(datadir)/alsa/sounds
 wav_files = Front_Left.wav \
 	Rear_Center.wav \
 	Rear_Right.wav \
diff -rup alsa-utils-1.0.28-orig/speaker-test/samples/Makefile.in alsa-utils-1.0.28/speaker-test/samples/Makefile.in
--- alsa-utils-1.0.28-orig/speaker-test/samples/Makefile.in	2014-08-01 22:14:26.022770937 +0300
+++ alsa-utils-1.0.28/speaker-test/samples/Makefile.in	2014-08-01 22:15:30.666769779 +0300
@@ -244,7 +244,7 @@ top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 udevrulesdir = @udevrulesdir@
 xmlto = @xmlto@
-sounddir = $(datadir)/sounds/alsa
+sounddir = $(datadir)/alsa/sounds
 wav_files = Front_Left.wav \
 	Rear_Center.wav \
 	Rear_Right.wav \
diff -rup alsa-utils-1.0.28-orig/speaker-test/speaker-test.1 alsa-utils-1.0.28/speaker-test/speaker-test.1
--- alsa-utils-1.0.28-orig/speaker-test/speaker-test.1	2014-08-01 22:14:26.016770937 +0300
+++ alsa-utils-1.0.28/speaker-test/speaker-test.1	2014-08-01 22:15:14.684770066 +0300
@@ -129,7 +129,7 @@ Use the given WAV file for the playback
 .TP
 \fB\-W\fP | \fB\-\-wavdir\fP \fIDIRECTORY\fP
 Specify the directory containing WAV files for playback.
-The default path is \fI/usr/share/sounds/alsa\fP.
+The default path is \fI/usr/share/alsa/sounds\fP.
 
 .TP
 \fB\-m\fP | \fB\-\-chmap\fP \fIMAP\fP
