diff -rup aterm-1.0.1-orig/src/main.c aterm-1.0.1/src/main.c
--- aterm-1.0.1-orig/src/main.c	2007-08-01 17:08:29.000000000 +0300
+++ aterm-1.0.1/src/main.c	2014-10-16 17:10:46.795851599 +0300
@@ -91,7 +91,7 @@ static XSizeHints szHint =
 
 static const char *def_colorName[] =
 {
-    "White", "Black", 		/* fg/bg */
+    "#d1e5ff", "#151525", 		/* fg/bg */
 /* low-intensity colors */
     "Black",			/* 0: black             (#000000) */
 #ifndef NO_BRIGHTCOLOR
