diff -rup bftpd-orig/Makefile bftpd/Makefile
--- bftpd-orig/Makefile	2014-03-23 14:03:40.510358502 -0400
+++ bftpd/Makefile	2014-03-23 14:09:27.094346248 -0400
@@ -3,11 +3,11 @@ VERSION=4.2
 CC?=gcc
 INSTALL=/usr/bin/install -c
 prefix=/usr
-CFLAGS=-g -O2 -DHAVE_CONFIG_H -Wall -I. -DVERSION=\"$(VERSION)\" -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -DPREFIX=\"$(prefix)\" 
+CFLAGS+=-DHAVE_CONFIG_H -Wall -I. -DVERSION=\"$(VERSION)\" -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -DPREFIX=\"$(prefix)\" 
 LIBS= -lcrypt
 mandir=${prefix}/man
-sbindir=${exec_prefix}/sbin
-DESTDIR=
+sbindir?=${exec_prefix}/sbin
+DESTDIR?=
 DIRPAX=
 PAX=
 
@@ -16,7 +16,7 @@ OBJS=bftpdutmp.o commands.o commands_adm
 SRCS=bftpdutmp.c commands.c commands_admin.c cwd.c dirlist.c list.c login.c logging.c main.c mystring.c options.c md5.c
 
 OBJS2LINK=$(OBJS) $(PAX)
-LDFLAGS=
+LDFLAGS?=
 
 all: bftpd
 
diff -rup bftpd-orig/Makefile.in bftpd/Makefile.in
--- bftpd-orig/Makefile.in	2014-03-23 14:03:40.510358502 -0400
+++ bftpd/Makefile.in	2014-03-23 14:09:30.302346135 -0400
@@ -15,7 +15,7 @@ OBJS=bftpdutmp.o commands.o commands_adm
 SRCS=bftpdutmp.c commands.c commands_admin.c cwd.c dirlist.c list.c login.c logging.c main.c mystring.c options.c md5.c
 
 OBJS2LINK=$(OBJS) $(PAX)
-LDFLAGS=
+LDFLAGS=@LDFLAGS@
 
 all: bftpd
 
@@ -26,13 +26,13 @@ bftpd: $(OBJS)
 $(OBJS): $(HEADERS) Makefile
 
 install: all
-	mkdir -p $(DESTDIR)/$(prefix)/sbin
-	$(INSTALL) -g 0 -m 700 -o 0 bftpd $(DESTDIR)/$(prefix)/sbin
+	mkdir -p $(DESTDIR)/${sbindir}
+	$(INSTALL) -m 700 bftpd $(DESTDIR)/${sbindir}
 	mkdir -p $(DESTDIR)/$(mandir)/man8
-	$(INSTALL) -g 0 -m 644 -o 0 bftpd.8 $(DESTDIR)/$(mandir)/man8
+	$(INSTALL) -m 644 bftpd.8 $(DESTDIR)/$(mandir)/man8
 	mkdir -p $(DESTDIR)/$(prefix)/etc
 	[ -f $(DESTDIR)/$(prefix)/etc/bftpd.conf ] || \
-		$(INSTALL) -g 0 -m 600 -o 0 bftpd.conf $(DESTDIR)/$(prefix)/etc
+		$(INSTALL) -m 600 bftpd.conf $(DESTDIR)/$(prefix)/etc
 	mkdir -p $(DESTDIR)/$(prefix)/var/log
 	touch $(DESTDIR)/$(prefix)/var/log/bftpd.log
 	chmod 644 $(DESTDIR)/$(prefix)/var/log/bftpd.log
