diff -rup conky-1.9.0-orig/data/conky.conf conky-1.9.0/data/conky.conf
--- conky-1.9.0-orig/data/conky.conf	2014-03-22 12:50:16.581956222 +0200
+++ conky-1.9.0/data/conky.conf	2014-03-22 12:52:02.327954328 +0200
@@ -34,8 +34,7 @@ draw_borders no
 draw_graph_borders yes
 draw_outline no
 draw_shades no
-use_xft yes
-xftfont DejaVu Sans Mono:size=12
+font *-terminus-medium-*-14-*
 gap_x 5
 gap_y 60
 minimum_size 5 5
