diff -rup dejavu-fonts-2.34-orig/Makefile dejavu-fonts-2.34/Makefile
--- dejavu-fonts-2.34-orig/Makefile	2013-12-26 22:42:43.790506929 +0000
+++ dejavu-fonts-2.34/Makefile	2013-12-26 22:42:54.698507220 +0000
@@ -77,10 +77,7 @@ $(TMPDIR)/%.sfd: $(SRCDIR)/%.sfd
 
 $(TMPDIR)/DejaVuLGC%.sfd: $(TMPDIR)/DejaVu%.sfd
 	@echo "[2] $< => $@"
-	sed -e 's,FontName: DejaVu,FontName: DejaVuLGC,'\
-	    -e 's,FullName: DejaVu,FullName: DejaVu LGC,'\
-	    -e 's,FamilyName: DejaVu,FamilyName: DejaVu LGC,'\
-            -e 's,"DejaVu \(\(Sans\|Serif\)*\( Condensed\| Mono\)*\( Bold\)*\( Oblique\|Italic\)*\)","DejaVu LGC \1",g' < $< > $@
+	cp -f $< $@
 	@echo "Stripping unwanted glyphs from $@"
 	$(LGC) $@
 	touch -r $< $@
