diff -rup dillo-master-orig/dillorc dillo-master/dillorc
--- dillo-master-orig/dillorc	2014-09-06 12:27:35.843089214 +0300
+++ dillo-master/dillorc	2014-09-06 12:36:03.517097051 +0300
@@ -60,17 +60,15 @@ w3c_plus_heuristics=YES
 #-------------------------------------------------------------------------
 
 # Set the start page.
-# Uncomment if you want to override the default start page.
-#start_page="file:/home/user/custom.html"
+# Comment if you want the default start page.
+start_page="file:/usr/share/doc/dslr/getting_started.html"
 
 # Set the home location
-home="http://www.dillo.org/"
+home="file:/usr/share/doc/dslr/getting_started.html"
 
 # Set search url to use with the search dialog.
 # %s is replaced with urlencoded keywords, and %% by '%'.
-search_url="http://www.google.com/search?q=%s"
-#search_url="http://search.lycos.com/default.asp?query=%s"
-#search_url="http://www.alltheweb.com/search?cat=web&query=%s"
+search_url="https://www.duckduckgo.com/html?q=%s"
 
 # Set the proxy information for http
 #http_proxy=http://localhost:8080/
@@ -94,7 +92,7 @@ search_url="http://www.google.com/search
 # Set the background color
 # bg_color=gray
 # bg_color=0xd6d6c0
-bg_color=0xdcd1ba
+bg_color=0xaaaaa7
 
 # Set the text color
 text_color=black
@@ -123,19 +121,19 @@ contrast_visited_color=YES
 # small  :  very nice! (it's "medium" without icon titles)
 # medium :  nice!
 # large  :  Traditional
-panel_size=medium
-small_icons=NO
+panel_size=tiny
+small_icons=YES
 
 # Here you can choose to hide some widgets of the dillo panel...
 #show_back=NO
 #show_forw=NO
-#show_home=NO   
+show_home=NO   
 #show_reload=NO
 #show_save=NO
 #show_stop=NO
 #show_bookmarks=NO
 #show_menubar=NO
-#show_clear_url=NO
+show_clear_url=NO
 #show_url=NO
 #show_search=NO
 #show_progress_box=NO
@@ -163,8 +161,8 @@ generate_submit=NO
 #-------------------------------------------------------------------------
 
 # Generic messsages (mainly for debugging specific parts)
-# Uncomment the following line to disable them.
-#show_msg=NO
+# Comment the following line to enable them.
+show_msg=NO
 
 # Soon we'll add the "show_debug_messages=NO" option...
 
