diff -rup dropbear-2014.66-orig/dropbear.8 dropbear-2014.66/dropbear.8
--- dropbear-2014.66-orig/dropbear.8	2014-10-24 18:02:26.731979124 +0300
+++ dropbear-2014.66/dropbear.8	2014-10-24 18:03:23.912978100 +0300
@@ -72,7 +72,7 @@ In program mode the \-F option is implie
 .TP
 .B \-P \fIpidfile
 Specify a pidfile to create when running as a daemon. If not specified, the 
-default is /var/run/dropbear.pid
+default is /run/dropbear.pid
 .TP
 .B \-a
 Allow remote hosts to connect to forwarded ports.
diff -rup dropbear-2014.66-orig/options.h dropbear-2014.66/options.h
--- dropbear-2014.66-orig/options.h	2014-10-24 18:02:26.731979124 +0300
+++ dropbear-2014.66/options.h	2014-10-24 18:03:04.184978453 +0300
@@ -258,13 +258,13 @@ much traffic. */
 /* The default file to store the daemon's process ID, for shutdown
    scripts etc. This can be overridden with the -P flag */
 #ifndef DROPBEAR_PIDFILE
-#define DROPBEAR_PIDFILE "/var/run/dropbear.pid"
+#define DROPBEAR_PIDFILE "/run/dropbear.pid"
 #endif
 
 /* The command to invoke for xauth when using X11 forwarding.
  * "-q" for quiet */
 #ifndef XAUTH_COMMAND
-#define XAUTH_COMMAND "/usr/bin/xauth -q"
+#define XAUTH_COMMAND "/bin/xauth -q"
 #endif
 
 /* if you want to enable running an sftp server (such as the one included with
@@ -276,7 +276,7 @@ much traffic. */
 
 /* This is used by the scp binary when used as a client binary. If you're
  * not using the Dropbear client, you'll need to change it */
-#define _PATH_SSH_PROGRAM "/usr/bin/dbclient"
+#define _PATH_SSH_PROGRAM "/bin/dbclient"
 
 /* Whether to log commands executed by a client. This only logs the 
  * (single) command sent to the server, not what a user did in a 
@@ -318,7 +318,7 @@ be overridden at runtime with -I. 0 disa
 #define DEFAULT_IDLE_TIMEOUT 0
 
 /* The default path. This will often get replaced by the shell */
-#define DEFAULT_PATH "/usr/bin:/bin"
+#define DEFAULT_PATH "/bin"
 
 /* Some other defines (that mostly should be left alone) are defined
  * in sysoptions.h */
diff -rup dropbear-2014.66-orig/sysoptions.h dropbear-2014.66/sysoptions.h
--- dropbear-2014.66-orig/sysoptions.h	2014-10-24 18:02:26.716979124 +0300
+++ dropbear-2014.66/sysoptions.h	2014-10-24 18:02:32.640979018 +0300
@@ -7,7 +7,7 @@
 #define DROPBEAR_VERSION "2014.66"
 #endif
 
-#define LOCAL_IDENT "SSH-2.0-dropbear_" DROPBEAR_VERSION
+#define LOCAL_IDENT "SSH-2.0-None"
 #define PROGNAME "dropbear"
 
 /* Spec recommends after one hour or 1 gigabyte of data. One hour
