diff -rup dropbear-2013.62-orig/Makefile.in dropbear-2013.62/Makefile.in
--- dropbear-2013.62-orig/Makefile.in	2014-02-01 15:10:07.230817215 +0200
+++ dropbear-2013.62/Makefile.in	2014-02-01 15:10:52.746818434 +0200
@@ -136,7 +136,7 @@ insmulti%: dropbearmulti
 	-rm -f $(DESTDIR)$(bindir)/$*$(EXEEXT) 
 	-ln -s $(bindir)/dropbearmulti$(EXEEXT) $(DESTDIR)$(bindir)/$*$(EXEEXT) 
 	$(INSTALL) -d $(DESTDIR)$(mandir)/man1
-	$(INSTALL) -m 644 $*.1  $(DESTDIR)$(mandir)/man1/$*.1
+	if test -e $*.1; then $(INSTALL) -m 644 $*.1 $(DESTDIR)$(mandir)/man1/$*.1; fi
 
 # dropbear should go in sbin, so it needs a seperate rule
 inst_dropbear: dropbear
