diff -rup dvtm-0.12-orig/vt.c dvtm-0.12/vt.c
--- dvtm-0.12-orig/vt.c	2014-07-05 10:32:09.000000000 +0300
+++ dvtm-0.12/vt.c	2014-09-24 14:24:13.883975002 +0300
@@ -84,7 +84,7 @@
 static bool is_utf8, has_default_colors;
 static short color_pairs_reserved, color_pairs_max, color_pair_current;
 static short *color2palette, default_fg, default_bg;
-static char vt_term[32] = "dvtm";
+static char vt_term[32] = "screen";
 
 typedef struct {
 	wchar_t *buf;
