diff -rup elvis-master-orig/draw.c elvis-master/draw.c
--- elvis-master-orig/draw.c	2014-10-10 11:20:50.055989182 +0300
+++ elvis-master/draw.c	2014-10-10 11:20:38.199989394 +0300
@@ -1114,7 +1114,7 @@ static void updateimage(win)
 			logicaltab = 0;
 		else
 		{
-			for (i = 0; di->newline[i].startrow < row && di->newline[i + 1].startrow - 1 < row; i++)
+			for (i = 0; di->newline[i].startrow < row && di->nlines > i && di->newline[i + 1].startrow - 1 < row; i++)
 			{
 			}
 			logicaltab = (row - di->newline[i].startrow) * ncols;
