diff -rup emelfm-master-orig/emelfm.c emelfm-master/emelfm.c
--- emelfm-master-orig/emelfm.c	2014-06-28 19:00:04.095963753 +0300
+++ emelfm-master/emelfm.c	2014-06-28 19:00:19.252963481 +0300
@@ -235,7 +235,7 @@ main(int argc, char *argv[])
     strncpy(cfg.output_font,
             "-*-courier-medium-r-normal-*-*-120-*-*-*-*-*",
             sizeof(cfg.output_font));
-    strncpy(cfg.xterm_command, "xterm", sizeof(cfg.xterm_command));
+    strncpy(cfg.xterm_command, "aterm", sizeof(cfg.xterm_command));
     chdir(getenv("HOME"));
     getcwd(app.left_view.dir, PATH_MAX);
     strncpy(app.right_view.dir, app.left_view.dir, PATH_MAX);
@@ -314,7 +314,7 @@ main(int argc, char *argv[])
     /* Setup the default toolbar */
     add_toolbar_button("df", "df", _("Available disk space"), FALSE);
     add_toolbar_button("free", "free", _("Memory information"), FALSE);
-    add_toolbar_button("X", "xterm &", _("Open an Xterm"), FALSE);
+    add_toolbar_button("X", "aterm &", _("Open an Xterm"), FALSE);
   }
 
   if (!read_keys_file())
