diff -rup emelfm-0.9.2-orig/Makefile.common emelfm-0.9.2/Makefile.common
--- emelfm-0.9.2-orig/Makefile.common	2014-06-28 18:19:05.153791493 +0300
+++ emelfm-0.9.2/Makefile.common	2014-06-28 18:19:10.524791397 +0300
@@ -1,13 +1,13 @@
-PREFIX = /usr/local
+PREFIX = 
 BIN_DIR = $(PREFIX)/bin
 LOCALEDIR = $(PREFIX)/share/locale
-PLUGINS_DIR = $(PREFIX)/share/emelfm/plugins
-DOC_DIR = $(PREFIX)/share/emelfm/docs
+PLUGINS_DIR = $(PREFIX)/lib/emelfm/plugins
+DOC_DIR = $(PREFIX)/usr/share/doc/emelfm
 
-NLS = -DENABLE_NLS
+NLS =
 GTK_INCLUDE = `gtk-config --cflags`
 GTK_LIB = `gtk-config --libs`
 GLIB_LIB = `glib-config --libs gthread`
-CC = gcc -O2 -Wall
+CC ?= 
 #CC = gcc -g -Wall
 
