diff -rup evilwm-1.1.0-orig/Makefile evilwm-1.1.0/Makefile
--- evilwm-1.1.0-orig/Makefile	2014-05-09 23:50:41.908715370 +0300
+++ evilwm-1.1.0/Makefile	2014-05-09 23:52:11.658713763 +0300
@@ -8,7 +8,7 @@ VPATH=$(src_dir)
 # Installation paths
 
 prefix = /usr
-bindir = $(prefix)/bin
+bindir = /bin
 datarootdir = $(prefix)/share
 mandir = $(datarootdir)/man
 man1dir = $(mandir)/man1
@@ -27,15 +27,15 @@ OPT_CPPFLAGS += -DINFOBANNER
 OPT_CPPFLAGS += -DMOUSE
 
 # Uncomment to support the Xrandr extension (thanks, Yura Semashko).
-OPT_CPPFLAGS += -DRANDR
-OPT_LDLIBS   += -lXrandr
+#OPT_CPPFLAGS += -DRANDR
+#OPT_LDLIBS   += -lXrandr
 
 # Uncomment to support shaped windows.
-OPT_CPPFLAGS += -DSHAPE
-OPT_LDLIBS   += -lXext
+#OPT_CPPFLAGS += -DSHAPE
+#OPT_LDLIBS   += -lXext
 
 # Uncomment to enable solid window drags.  This can be slow on old systems.
-OPT_CPPFLAGS += -DSOLIDDRAG
+#OPT_CPPFLAGS += -DSOLIDDRAG
 
 # Uncomment to compile in certain text messages like help.  Recommended.
 OPT_CPPFLAGS += -DSTDIO
@@ -74,10 +74,10 @@ OPT_CPPFLAGS += -DWARP_POINTER
 # Build tools
 
 # Change this if you don't use gcc:
-CC = gcc
+CC ?= gcc
 
 # Override if desired:
-CFLAGS = -Os -std=c99
+CFLAGS ?= -Os -std=c99
 WARN = -Wall -W -Wstrict-prototypes -Wpointer-arith -Wcast-align \
 	-Wshadow -Waggregate-return -Wnested-externs -Winline -Wwrite-strings \
 	-Wundef -Wsign-compare -Wmissing-prototypes -Wredundant-decls
