diff -rup font-adobe-75dpi-1.0.3-orig/Makefile.in font-adobe-75dpi-1.0.3/Makefile.in
--- font-adobe-75dpi-1.0.3-orig/Makefile.in	2014-10-17 12:31:51.021897977 +0300
+++ font-adobe-75dpi-1.0.3/Makefile.in	2014-10-17 12:32:02.145897777 +0300
@@ -668,7 +668,6 @@ info-am:
 
 install-data-am: install-fontDATA
 	@$(NORMAL_INSTALL)
-	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
 install-dvi: install-dvi-am
 
 install-dvi-am:
