diff -rup gcalendar-0.6.0-orig/gtkcalendar.c gcalendar-0.6.0/gtkcalendar.c
--- gcalendar-0.6.0-orig/gtkcalendar.c	2014-06-28 16:58:57.955877581 +0300
+++ gcalendar-0.6.0/gtkcalendar.c	2014-06-28 16:59:36.238876895 +0300
@@ -27,7 +27,7 @@
 #include "gtkcalendar.h"
 #include "lib_date.h"
 
-#define HELVETICA_10_FONT  "-adobe-helvetica-bold-r-normal-*-10-*-*-*-*-*-*-*"
+#define HELVETICA_10_FONT  "-*-lucida-bold-r-*-*-10-*-*-*-*-*-*-*"
 #define FIXED_10_FONT      "-b&h-lucidatypewriter-medium-r-normal-*-10-*-*-*-*-*-*-*"
 
 
