diff -rup gcolor-0.4-orig/src/interface.c gcolor-0.4/src/interface.c
--- gcolor-0.4-orig/src/interface.c	2014-04-08 17:38:32.867972853 +0300
+++ gcolor-0.4/src/interface.c	2014-04-08 17:39:14.476972108 +0300
@@ -41,9 +41,9 @@ create_window1 (void)
   GtkWidget *listnamelabel;
 
   window1 = gtk_window_new (GTK_WINDOW_TOPLEVEL);
-  gtk_widget_set_name (window1, "window1");
-  gtk_object_set_data (GTK_OBJECT (window1), "window1", window1);
-  gtk_window_set_title (GTK_WINDOW (window1), _("window1"));
+  gtk_widget_set_name (window1, "gcolor");
+  gtk_object_set_data (GTK_OBJECT (window1), "gcolor", window1);
+  gtk_window_set_title (GTK_WINDOW (window1), _("gcolor"));
 
   vbox1 = gtk_vbox_new (FALSE, 0);
   gtk_widget_set_name (vbox1, "vbox1");
