diff -rup gdk-pixbuf-0.22.0-orig/gdk-pixbuf-config.in gdk-pixbuf-0.22.0/gdk-pixbuf-config.in
--- gdk-pixbuf-0.22.0-orig/gdk-pixbuf-config.in	2014-01-10 11:12:53.253649954 +0000
+++ gdk-pixbuf-0.22.0/gdk-pixbuf-config.in	2014-01-10 11:13:28.329650895 +0000
@@ -44,7 +44,7 @@ while test $# -gt 0; do
       ;;
     --libs)
       libdirs=-L@libdir@
-      echo $libdirs -lgdk_pixbuf `gtk-config --libs`
+      echo $libdirs -lgdk_pixbuf @STATIC_LIB_DEPS@ `gtk-config --libs`
       ;;
     *)
       echo "${usage}" 1>&2
