diff -rup gnu-efi-3.0-orig/Make.defaults gnu-efi-3.0/Make.defaults
--- gnu-efi-3.0-orig/Make.defaults	2014-08-23 12:04:57.762957705 +0300
+++ gnu-efi-3.0/Make.defaults	2014-08-23 12:26:56.645934086 +0300
@@ -52,10 +52,10 @@ INSTALL	     := install
 
 # Compilation tools
 HOSTCC       := $(prefix)gcc
-CC           := $(prefix)$(CROSS_COMPILE)gcc
+CC           ?= $(prefix)$(CROSS_COMPILE)gcc
 AS           := $(prefix)$(CROSS_COMPILE)as
 LD           := $(prefix)$(CROSS_COMPILE)ld
-AR           := $(prefix)$(CROSS_COMPILE)ar
+AR           ?= $(prefix)$(CROSS_COMPILE)ar
 RANLIB       := $(prefix)$(CROSS_COMPILE)ranlib
 OBJCOPY      := $(prefix)$(CROSS_COMPILE)objcopy
 
