diff -rup gpm-1.20.7-orig/src/Makefile.in gpm-1.20.7/src/Makefile.in
--- gpm-1.20.7-orig/src/Makefile.in	2012-10-26 23:21:38.000000000 +0200
+++ gpm-1.20.7/src/Makefile.in	2014-11-08 13:34:45.171940828 +0200
@@ -79,7 +79,7 @@ prog/%:	prog/%.o
 #		| $(SED) '\''s/\($*\)\.o\([ :]*\)/\1.o \1.lo\2/g'\'' > $(DEPDIR)/$@'
 
 # Do it all!
-all:	gpm lib/libgpm.so.@abi_lev@ @LIBGPM_A@ $(PROG)
+all:	gpm @LIBGPM_A@ $(PROG)
 
 gpm:	$(GOBJ)
 	$(CC) @LDFLAGS@ $(LDFLAGS) -o $@ $(GOBJ) @LIBS@ $(LIBS) -lm
@@ -149,14 +149,14 @@ prog/gpm-root.c:	$(srcdir)/prog/gpm-root
 	$(YACC) $(YFLAGS) $< && mv y.tab.c prog/gpm-root.c
 
 # gpm-root needs an own rule, because gpm-root.c is not in $(srcdir)
-prog/gpm-root: prog/gpm-root.c lib/libgpm.so.@abi_lev@
+prog/gpm-root: prog/gpm-root.c @LIBGPM_A@
 	$(CC) -I. @CPPFLAGS@ $(CPPFLAGS) @CFLAGS@ $(CFLAGS) -c -o $@.o $<
-	$(CC) @LDFLAGS@ $(LDFLAGS) -o $@ $@.o @LIBS@ $(LIBS) lib/libgpm.so.@abi_lev@
+	$(CC) @LDFLAGS@ $(LDFLAGS) -o $@ $@.o @LIBS@ $(LIBS) @LIBGPM_A@
 
 prog/mouse-test:	prog/mouse-test.o mice.o twiddler.o synaptics.o prog/open_console.o
 	$(CC) @LDFLAGS@ $(LDFLAGS) -o $@ $^ @LIBS@ $(LIBS) -lm
 
-$(PROG):	lib/libgpm.so.@abi_lev@
+$(PROG):	@LIBGPM_A@
 
 # Subdirectory lib/
 lib/libgpm.a:	$(LOBJ)
