--- gdk/gdkim.c	2001-04-02 10:22:18.000000000 +0800
+++ gdk/gdkim.c.new	2012-07-04 19:18:33.000000000 +0800
@@ -112,12 +112,12 @@ gdk_set_locale (void)
   
   if (!XSupportsLocale ())
     {
-      g_warning ("locale not supported by Xlib, locale set to C");
+      /*g_warning ("locale not supported by Xlib, locale set to C");*/
       setlocale (LC_ALL, "C");
     }
   
-  if (!XSetLocaleModifiers (""))
-    g_warning ("can not set locale modifiers");
+  /*if (!XSetLocaleModifiers (""))
+    g_warning ("can not set locale modifiers");*/
 
   current_locale = setlocale (LC_ALL, NULL);
 
--- gdk/gdkwindow.c	2001-02-23 06:02:11.000000000 +0800
+++ gdk/gdkwindow.c.new	2012-07-04 19:27:27.000000000 +0800
@@ -478,7 +478,17 @@ gdk_window_new (GdkWindow     *parent,
                       title, title,
                       NULL, 0,
                       NULL, NULL, NULL);
-  
+  //tinyX11 hack
+  XTextProperty wname, iname;
+  if (XStringListToTextProperty(& title, 1, & wname) == 0)
+		printf ("failed to set window name structure");
+		
+  if (XStringListToTextProperty(& title, 1, & iname) == 0)
+		printf ("failed to set icon name structure");
+		
+  XSetWMName(private->xdisplay, private->xwindow, &wname);
+  XSetWMIconName(private->xdisplay, private->xwindow, &iname);
+  //          
   if (attributes_mask & GDK_WA_WMCLASS)
     {
       class_hint = XAllocClassHint ();
@@ -1186,6 +1196,18 @@ gdk_window_set_title (GdkWindow   *windo
   if (!private->destroyed)
     XmbSetWMProperties (private->xdisplay, private->xwindow,
 			title, title, NULL, 0, NULL, NULL, NULL);
+  
+  //tinyX11 hack
+  XTextProperty wname, iname;
+  if (XStringListToTextProperty(& title, 1, & wname) == 0)
+		printf ("failed to set window name structure");
+		
+  if (XStringListToTextProperty(& title, 1, & iname) == 0)
+	printf ("failed to set icon name structure");
+		
+  XSetWMName(private->xdisplay, private->xwindow, &wname);
+  XSetWMIconName(private->xdisplay, private->xwindow, &iname);
+  //	
 }
 
 void          
--- ./gdk/gdkwindow.c.01	2013-03-04 17:57:58.000000000 +0800
+++ ./gdk/gdkwindow.c	2013-03-04 19:37:53.000000000 +0800
@@ -2105,14 +2105,29 @@
   if (res < 0)
     {
       g_warning ("Error converting icon name to text property: %d\n", res);
-      return;
+      //return;	//tinyX11 hack
     }
-  
-  XSetWMIconName (window_private->xdisplay, window_private->xwindow,
+	//tinyX11 hack
+	XTextProperty iname;
+ 
+	if (XStringListToTextProperty(& name, 1, & iname) == 0) {
+		printf ("failed to set icon name structure\n");
+	} else {
+		XSetWMIconName (window_private->xdisplay, window_private->xwindow,
+		  &iname);
+		printf ("but succeded to do it anyway...\n");
+  	}
+	
+	if (iname.value)
+    	XFree (iname.value);
+	//	
+ 	   
+ /* XSetWMIconName (window_private->xdisplay, window_private->xwindow,
 		  &property);
   
   if (property.value)
     XFree (property.value);
+ */
 }
 
 void          
--- ./gdk/gdkim.c.01	2013-03-08 15:54:59.000000000 +0800
+++ ./gdk/gdkim.c	2013-03-08 16:22:34.000000000 +0800
@@ -106,7 +106,7 @@
   gchar *current_locale;
 
   gdk_use_mb = FALSE;
-
+  if (! strcmp (setlocale (LC_ALL, NULL), "C.UTF-8")) {return; } //musl fix
   if (!setlocale (LC_ALL,""))
     g_warning ("locale not supported by C library");
   
--- ./gdk/gdkselection.c.01	2013-03-08 16:33:01.000000000 +0800
+++ ./gdk/gdkselection.c	2013-03-08 16:33:47.000000000 +0800
@@ -327,9 +327,9 @@
       property.value = NULL;
       property.nitems = 0;
     }
-
+#if 0	//musl fix or tinyxlib fixme...
   g_assert (property.encoding == gdk_atom_intern ("COMPOUND_TEXT", FALSE) && property.format == 8);
-
+#endif
   if (encoding)
     *encoding = property.encoding;
   if (format)
