diff -rup gtk+-1.2.10-orig/gtk/gtkfilesel.c gtk+-1.2.10/gtk/gtkfilesel.c
--- gtk+-1.2.10-orig/gtk/gtkfilesel.c	2014-03-29 23:52:56.832940673 +0300
+++ gtk+-1.2.10/gtk/gtkfilesel.c	2014-03-29 23:53:53.915939651 +0300
@@ -2025,7 +2025,12 @@ open_new_dir(gchar* dir_name, struct sta
 
   buffer_ptr = sent->name_buffer;
 
-  rewinddir(directory);
+  closedir(directory);
+  directory = opendir(dir_name);
+  if (!directory) {
+    cmpl_errno = errno;
+    return NULL;
+  }
 
   for(i = 0; i < entry_count; i += 1)
     {
