diff -rup gtk+-1.2.10-orig/gtk/gtkstyle.c gtk+-1.2.10/gtk/gtkstyle.c
--- gtk+-1.2.10-orig/gtk/gtkstyle.c	2014-04-04 18:36:19.155971684 +0300
+++ gtk+-1.2.10/gtk/gtkstyle.c	2014-04-04 18:37:13.798970705 +0300
@@ -398,7 +398,7 @@ gtk_style_new (void)
   if (!default_font)
     {
       default_font =
-	gdk_font_load ("-adobe-helvetica-medium-r-normal--*-120-*-*-*-*-iso8859-1");
+	gdk_font_load ("*-terminus-medium-*-14-*");
       if (!default_font)
 	default_font = gdk_font_load ("fixed");
       if (!default_font)
