diff -rup ircii-20131230-orig/Makefile.in ircii-20131230/Makefile.in
--- ircii-20131230-orig/Makefile.in	2014-02-13 21:07:54.004190523 +0200
+++ ircii-20131230/Makefile.in	2014-02-13 21:08:14.620191073 +0200
@@ -182,8 +182,8 @@ installwserv: wserv installdirs
 
 installhelp:
 	$(INSTALL_HELP_CMD)
-	find $(HELP_DIR) -type d -print | xargs chmod a+rx
-	find $(HELP_DIR) -type f -print | xargs chmod a+r
+	find $(HELP_DIR) -type d -print | xargs -I'{}' chmod a+rx '{}'
+	find $(HELP_DIR) -type f -print | xargs -I'{}' chmod a+r '{}'
 
 installman:
 	(cd $(topdir)/doc; for i in ircII.1 ircbug.1; do $(INSTALL_DATA) $$i $(mandir); done)
