diff -rup jwm-2.2.2-orig/example.jwmrc jwm-2.2.2/example.jwmrc
--- jwm-2.2.2-orig/example.jwmrc	2014-05-09 19:14:01.536999936 +0300
+++ jwm-2.2.2/example.jwmrc	2014-05-09 19:14:16.729999664 +0300
@@ -3,116 +3,83 @@
 <JWM>
 
     <!-- The root menu. -->
-    <RootMenu onroot="12">
-        <Program icon="terminal.png" label="Terminal">xterm</Program>
-        <Menu icon="folder.png" label="Applications">
-            <Program icon="music.png" label="Audacious">audacious</Program>
-            <Program icon="calculator.png" label="Calculator">xcalc</Program>
-            <Program icon="gimp.png" label="Gimp">gimp</Program>
-            <Program icon="www.png" label="Iceweasel">iceweasel</Program>
-            <Program icon="editor.png" label="XEdit">xedit</Program>
-        </Menu>
-        <Menu icon="folder.png" label="Utilities">
-            <Program icon="font.png" label="Fonts">xfontsel</Program>
-            <Program icon="window.png" label="Window Properties">
-                xprop | xmessage -file -
-            </Program>
-        </Menu>
+    <RootMenu height="32">
+        <Program label="Execute...">grun</Program>
+        <Program label="Find...">gtkfind</Program>
+        <Program label="Browse Files">rox</Program>
+        <Program label="Terminal">aterm -title Terminal -e screen</Program>
         <Separator/>
-        <Program icon="lock.png" label="Lock">
-            xscreensaver-command -lock
-        </Program>
+        <Desktops/>
         <Separator/>
-        <Restart label="Restart" icon="restart.png"/>
-        <Exit label="Exit" confirm="true" icon="quit.png"/>
+        <Program label="Suspend">suspend</Program>
+        <Restart label="Restart"/>
+        <Exit label="Exit"/>
     </RootMenu>
 
-    <!-- Options for program groups. -->
-    <Group>
-        <Class>Pidgin</Class>
-        <Option>sticky</Option>
-    </Group>
-    <Group>
-        <Name>xterm</Name>
-        <Option>vmax</Option>
-        <Option>icon:terminal.png</Option>
-    </Group>
-    <Group>
-        <Name>xedit</Name>
-        <Option>icon:editor.png</Option>
-    </Group>
-    <Group>
-        <Name>xcalc</Name>
-        <Option>icon:calculator.png</Option>
-    </Group>
-
     <!-- Tray at the bottom. -->
-    <Tray x="0" y="-1" height="32" autohide="false">
-
-        <TrayButton label="JWM">root:1</TrayButton>
+    <Tray y="-1" height="32" border="1">
+        <TrayButton label="Menu">root:1</TrayButton>
         <TrayButton label="_">showdesktop</TrayButton>
-
-        <Pager labeled="true"/>
-
-        <TaskList maxwidth="256"/>
-
+        <TrayButton label="*">exec:grun</TrayButton>
+        <Pager/>
+        <TaskList/>
         <Dock/>
-        <Clock>xclock</Clock>
-
+        <Clock format="%H:%M">gcalendar</Clock>
     </Tray>
 
     <!-- Visual Styles -->
     <WindowStyle>
-        <Font>FreeSans-9:bold</Font>
-        <Width>4</Width>
-        <Height>20</Height>
+        <Font>*-fixed-medium-*-14-*</Font>
+        <Width>3</Width>
+        <Height>22</Height>
         <Active>
-            <Text>#FFFFFF</Text>
-            <Title>#CC7700:#884400</Title>
-            <Outline>#000000</Outline>
-            <Opacity>1.0</Opacity>
+            <Text>#111111</Text>
+            <Title>#999999</Title>
+            <Outline>#333333</Outline>
         </Active>
         <Inactive>
-            <Text>#FFFFFF</Text>
-            <Title>#333333:#111111</Title>
-            <Outline>#000000</Outline>
-            <Opacity>0.5:0.9:0.1</Opacity>
+            <Text>#333333</Text>
+            <Title>#b6b8b0</Title>
+            <Outline>#333333</Outline>
         </Inactive>
     </WindowStyle>
     <TrayStyle>
-        <Font>FreeSans-12:bold</Font>
-        <Background>#111111</Background>
-        <Foreground>#FFFFFF</Foreground>
-        <Opacity>0.75</Opacity>
+        <Font>*-fixed-medium-*-14-*</Font>
+        <Background>#b6b8b0</Background>
+        <Foreground>#111111</Foreground>
     </TrayStyle>
+    <TrayButtonStyle>
+        <Font>*-fixed-medium-*-14-*</Font>
+        <Foreground>#111111</Foreground>
+        <Background>#b6b8b0:#dddddd</Background>
+        <ActiveForeground>black</ActiveForeground>
+        <ActiveBackground>#FFFFFF:#A4A4A4</ActiveBackground>
+    </TrayButtonStyle>
     <TaskListStyle>
-        <Font>FreeSans-12:bold</Font>
-        <ActiveForeground>#FFFFFF</ActiveForeground>
-        <ActiveBackground>#111111:#333333</ActiveBackground>
-        <Foreground>#FFFFFF</Foreground>
-        <Background>#333333:#111111</Background>
+        <Font>*-fixed-medium-*-14-*</Font>
+        <ActiveForeground>#111111</ActiveForeground>
+        <ActiveBackground>#999999:#cccccc</ActiveBackground>
+	<Background>#b6b8b0:#dddddd</Background>
     </TaskListStyle>
     <PagerStyle>
-        <Outline>#000000</Outline>
-        <Text>#FFFFFF</Text>
-        <Foreground>#444444</Foreground>
-        <Background>#111111</Background>
-        <ActiveForeground>#CC7700</ActiveForeground>
-        <ActiveBackground>#884400</ActiveBackground>
+       <Outline>black</Outline>
+       <Foreground>#333333</Foreground>
+       <Background>#2A2936</Background>
+       <ActiveForeground>#666666</ActiveForeground>
+       <ActiveBackground>#263A60</ActiveBackground>
     </PagerStyle>
     <MenuStyle>
-        <Font>FreeSans-11</Font>
-        <Foreground>#FFFFFF</Foreground>
-        <Background>#333333</Background>
-        <ActiveForeground>#FFFFFF</ActiveForeground>
-        <ActiveBackground>#CC7700:#884400</ActiveBackground>
-        <Opacity>0.85</Opacity>
+        <Font>*-fixed-medium-*-14-*</Font>
+        <Foreground>#111111</Foreground>
+        <Background>#DCDAD5</Background>
+        <ActiveForeground>#111111</ActiveForeground>
+        <ActiveBackground>#b6b8b0:#dddddd</ActiveBackground>
     </MenuStyle>
     <PopupStyle>
-        <Font>FreeSans-10</Font>
-        <Outline>#000000</Outline>
-        <Foreground>#000000</Foreground>
-        <Background>#999999</Background>
+        <Font>*-fixed-medium-*-14-*</Font>
+        <Outline>#333333</Outline>
+        <Foreground>#111111</Foreground>
+        <Background>#b6b8b0</Background>
     </PopupStyle>
 
     <!-- Path where icons can be found.
@@ -120,7 +87,7 @@
          for icons in multiple paths.
       -->
     <IconPath>
-        /usr/share/icons/wm-icons/32x32-gant
+        /usr/share/pixmaps
     </IconPath>
 
     <!-- Virtual Desktops -->
@@ -130,7 +97,7 @@
               contained within a Desktop tag to give a specific background
               for that desktop.
          -->
-        <Background type="image">$HOME/images/bg.jpg</Background>
+        <Background type="gradient">#263a44:#263a60</Background>
     </Desktops>
 
     <!-- Double click speed (in milliseconds) -->
@@ -140,16 +107,19 @@
     <DoubleClickDelta>2</DoubleClickDelta>
 
     <!-- The focus model (sloppy or click) -->
-    <FocusModel>sloppy</FocusModel>
+    <FocusModel>click</FocusModel>
 
     <!-- The snap mode (none, screen, or border) -->
-    <SnapMode distance="10">border</SnapMode>
+    <SnapMode distance="10">screen</SnapMode>
 
     <!-- The move mode (outline or opaque) -->
-    <MoveMode>opaque</MoveMode>
+    <MoveMode coordinates="window">outline</MoveMode>
 
     <!-- The resize mode (outline or opaque) -->
-    <ResizeMode>opaque</ResizeMode>
+    <ResizeMode coordinates="window">outline</ResizeMode>
+
+    <!-- Startup commands -->
+    <StartupCommand>conky</StartupCommand>
 
     <!-- Key bindings -->
     <Key key="Up">up</Key>
@@ -167,11 +137,17 @@
     <Key mask="A" key="F4">close</Key>
     <Key mask="A" key="#">desktop#</Key>
     <Key mask="A" key="F1">root:1</Key>
-    <Key mask="A" key="F2">window</Key>
+    <Key mask="A" key="F2">exec:grun</Key>
     <Key mask="A" key="F10">maximize</Key>
     <Key mask="A" key="Right">rdesktop</Key>
     <Key mask="A" key="Left">ldesktop</Key>
     <Key mask="A" key="Up">udesktop</Key>
     <Key mask="A" key="Down">ddesktop</Key>
 
+    <Key mask="4" key="Up">maximize</Key>
+    <Key mask="4" key="Down">minimize</Key>
+    <Key mask="4" key="d">showdesktop</Key>
+    <Key mask="4" key="r">exec:grun</Key>
+    <Key mask="4" key="t">exec:aterm -title Terminal -e screen</Key>
+    <Key mask="A" key="Return">exec:aterm -title Terminal -e screen</Key>
 </JWM>
