diff -rup jwm-2.2.2-orig/example.jwmrc jwm-2.2.2/example.jwmrc
--- jwm-2.2.2-orig/example.jwmrc	2014-10-14 14:27:52.968847065 +0300
+++ jwm-2.2.2/example.jwmrc	2014-10-15 21:48:51.400905205 +0300
@@ -2,176 +2,167 @@
 
 <JWM>
 
-    <!-- The root menu. -->
-    <RootMenu onroot="12">
-        <Program icon="terminal.png" label="Terminal">xterm</Program>
-        <Menu icon="folder.png" label="Applications">
-            <Program icon="music.png" label="Audacious">audacious</Program>
-            <Program icon="calculator.png" label="Calculator">xcalc</Program>
-            <Program icon="gimp.png" label="Gimp">gimp</Program>
-            <Program icon="www.png" label="Iceweasel">iceweasel</Program>
-            <Program icon="editor.png" label="XEdit">xedit</Program>
-        </Menu>
-        <Menu icon="folder.png" label="Utilities">
-            <Program icon="font.png" label="Fonts">xfontsel</Program>
-            <Program icon="window.png" label="Window Properties">
-                xprop | xmessage -file -
-            </Program>
-        </Menu>
-        <Separator/>
-        <Program icon="lock.png" label="Lock">
-            xscreensaver-command -lock
-        </Program>
-        <Separator/>
-        <Restart label="Restart" icon="restart.png"/>
-        <Exit label="Exit" confirm="true" icon="quit.png"/>
-    </RootMenu>
-
-    <!-- Options for program groups. -->
-    <Group>
-        <Class>Pidgin</Class>
-        <Option>sticky</Option>
-    </Group>
-    <Group>
-        <Name>xterm</Name>
-        <Option>vmax</Option>
-        <Option>icon:terminal.png</Option>
-    </Group>
-    <Group>
-        <Name>xedit</Name>
-        <Option>icon:editor.png</Option>
-    </Group>
-    <Group>
-        <Name>xcalc</Name>
-        <Option>icon:calculator.png</Option>
-    </Group>
-
-    <!-- Tray at the bottom. -->
-    <Tray x="0" y="-1" height="32" autohide="false">
-
-        <TrayButton label="JWM">root:1</TrayButton>
-        <TrayButton label="_">showdesktop</TrayButton>
-
-        <Pager labeled="true"/>
-
-        <TaskList maxwidth="256"/>
-
-        <Dock/>
-        <Clock>xclock</Clock>
-
-    </Tray>
-
-    <!-- Visual Styles -->
-    <WindowStyle>
-        <Font>FreeSans-9:bold</Font>
-        <Width>4</Width>
-        <Height>20</Height>
-        <Active>
-            <Text>#FFFFFF</Text>
-            <Title>#CC7700:#884400</Title>
-            <Outline>#000000</Outline>
-            <Opacity>1.0</Opacity>
-        </Active>
-        <Inactive>
-            <Text>#FFFFFF</Text>
-            <Title>#333333:#111111</Title>
-            <Outline>#000000</Outline>
-            <Opacity>0.5:0.9:0.1</Opacity>
-        </Inactive>
-    </WindowStyle>
-    <TrayStyle>
-        <Font>FreeSans-12:bold</Font>
-        <Background>#111111</Background>
-        <Foreground>#FFFFFF</Foreground>
-        <Opacity>0.75</Opacity>
-    </TrayStyle>
-    <TaskListStyle>
-        <Font>FreeSans-12:bold</Font>
-        <ActiveForeground>#FFFFFF</ActiveForeground>
-        <ActiveBackground>#111111:#333333</ActiveBackground>
-        <Foreground>#FFFFFF</Foreground>
-        <Background>#333333:#111111</Background>
-    </TaskListStyle>
-    <PagerStyle>
-        <Outline>#000000</Outline>
-        <Text>#FFFFFF</Text>
-        <Foreground>#444444</Foreground>
-        <Background>#111111</Background>
-        <ActiveForeground>#CC7700</ActiveForeground>
-        <ActiveBackground>#884400</ActiveBackground>
-    </PagerStyle>
-    <MenuStyle>
-        <Font>FreeSans-11</Font>
-        <Foreground>#FFFFFF</Foreground>
-        <Background>#333333</Background>
-        <ActiveForeground>#FFFFFF</ActiveForeground>
-        <ActiveBackground>#CC7700:#884400</ActiveBackground>
-        <Opacity>0.85</Opacity>
-    </MenuStyle>
-    <PopupStyle>
-        <Font>FreeSans-10</Font>
-        <Outline>#000000</Outline>
-        <Foreground>#000000</Foreground>
-        <Background>#999999</Background>
-    </PopupStyle>
-
-    <!-- Path where icons can be found.
-         IconPath can be listed multiple times to allow searching
-         for icons in multiple paths.
-      -->
-    <IconPath>
-        /usr/share/icons/wm-icons/32x32-gant
-    </IconPath>
-
-    <!-- Virtual Desktops -->
-    <!-- Desktop tags can be contained within Desktops for desktop names. -->
-    <Desktops width="2" height="1">
-        <!-- Default background. Note that a Background tag can be
-              contained within a Desktop tag to give a specific background
-              for that desktop.
-         -->
-        <Background type="image">$HOME/images/bg.jpg</Background>
-    </Desktops>
-
-    <!-- Double click speed (in milliseconds) -->
-    <DoubleClickSpeed>400</DoubleClickSpeed>
-
-    <!-- Double click delta (in pixels) -->
-    <DoubleClickDelta>2</DoubleClickDelta>
-
-    <!-- The focus model (sloppy or click) -->
-    <FocusModel>sloppy</FocusModel>
-
-    <!-- The snap mode (none, screen, or border) -->
-    <SnapMode distance="10">border</SnapMode>
-
-    <!-- The move mode (outline or opaque) -->
-    <MoveMode>opaque</MoveMode>
-
-    <!-- The resize mode (outline or opaque) -->
-    <ResizeMode>opaque</ResizeMode>
-
-    <!-- Key bindings -->
-    <Key key="Up">up</Key>
-    <Key key="Down">down</Key>
-    <Key key="Right">right</Key>
-    <Key key="Left">left</Key>
-    <Key key="h">left</Key>
-    <Key key="j">down</Key>
-    <Key key="k">up</Key>
-    <Key key="l">right</Key>
-    <Key key="Return">select</Key>
-    <Key key="Escape">escape</Key>
-
-    <Key mask="A" key="Tab">nextstacked</Key>
-    <Key mask="A" key="F4">close</Key>
-    <Key mask="A" key="#">desktop#</Key>
-    <Key mask="A" key="F1">root:1</Key>
-    <Key mask="A" key="F2">window</Key>
-    <Key mask="A" key="F10">maximize</Key>
-    <Key mask="A" key="Right">rdesktop</Key>
-    <Key mask="A" key="Left">ldesktop</Key>
-    <Key mask="A" key="Up">udesktop</Key>
-    <Key mask="A" key="Down">ddesktop</Key>
-
+	<!-- The root menu. -->
+	<RootMenu onroot="3">
+		<Desktops/>
+		<Separator/>
+		<Restart label="Restart"/>
+		<Exit label="Exit to prompt"/>
+		<Program label="Suspend">suspend</Program>
+		<Program label="Reboot">reboot</Program>
+		<Program label="Power off">poweroff</Program>
+	</RootMenu>
+
+	<!-- Tray at the bottom. -->
+	<Tray y="-1" height="48" border="1" halign="center" autohide="true">
+		<TrayButton icon="rlsd.xpm" border="false">root:3</TrayButton>
+		<TrayButton icon="text.xpm" popup="Text editor" border="false">exec:gtkedit</TrayButton>
+		<TrayButton icon="settings.xpm" popup="Settings" border="false">exec:settings</TrayButton>
+		<TrayButton icon="terminal.xpm" popup="Terminal" border="false">exec:aterm</TrayButton>
+		<TrayButton icon="home.xpm" popup="File manager" border="false">exec:rox</TrayButton>
+		<TrayButton icon="executable.xpm" popup="Launcher" border="false">exec:grun</TrayButton>
+		<TrayButton icon="chat.xpm" popup="Chat" border="false">exec:aterm -title ScrollZ -e scrollz -c \#rlsdlinux irc.freenode.net</TrayButton>
+		<TrayButton icon="help.png" popup="Help" border="false">exec:dillo /usr/share/doc/rlsd/getting_started.html</TrayButton>
+	</Tray>
+
+	<!-- Application-specific options -->
+	<Group>
+		<Name>xhippo</Name>
+		<Option>sticky</Option>
+		<Option>layer:above</Option>
+	</Group>
+
+	<Group>
+		<Name>aterm</Name>
+		<Option>noborder</Option>
+	</Group>
+
+	<Group>
+		<Name>ffplay</Name>
+		<Option>noborder</Option>
+	</Group>
+
+	<!-- Visual Styles -->
+	<WindowStyle>
+		<Font>-adobe-helvetica-bold-r-normal-*-12-*-*-*-p-*-iso8859-1</Font>
+		<Width>3</Width>
+		<Height>20</Height>
+		<Active>
+			<Text>black</Text>
+			<Title>#aaaaaa:#808080</Title>
+			<Outline>#555555</Outline>
+		</Active>
+		<Inactive>
+			<Text>white</Text>
+			<Title>#555555:#303030</Title>
+			<Outline>#777777</Outline>
+		</Inactive>
+	</WindowStyle>
+	<TrayStyle>
+		<Font>-adobe-helvetica-bold-r-normal-*-12-*-*-*-p-*-iso8859-1</Font>
+		<Background>#333333:#555555</Background>
+		<Foreground>white</Foreground>
+	</TrayStyle>
+	<TrayButtonStyle>
+		<Foreground>white</Foreground>
+		<Background>#444444:#777777</Background>
+		<ActiveForeground>black</ActiveForeground>
+		<ActiveBackground>#666666:#bbbbbb</ActiveBackground>
+	</TrayButtonStyle>
+	<TaskListStyle>
+		<ActiveForeground>black</ActiveForeground>
+		<ActiveBackground>#666666:#bbbbbb</ActiveBackground>
+		<Foreground>white</Foreground>
+		<Background>#444444:#777777</Background>
+	</TaskListStyle>
+	<PagerStyle>
+		<Outline>#222222</Outline>
+		<Foreground>#666666</Foreground>
+		<Background>#3b3b3b</Background>
+		<ActiveForeground>#999999</ActiveForeground>
+		<ActiveBackground>#666666</ActiveBackground>
+	</PagerStyle>
+	<MenuStyle>
+		<Font>-adobe-helvetica-bold-r-normal-*-12-*-*-*-p-*-iso8859-1</Font>
+		<Foreground>white</Foreground>
+		<Background>#333333</Background>
+		<ActiveForeground>black</ActiveForeground>
+		<ActiveBackground>#666666:#bbbbbb</ActiveBackground>
+	</MenuStyle>
+	<PopupStyle>
+		<Font>-adobe-helvetica-bold-r-normal-*-12-*-*-*-p-*-iso8859-1</Font>
+		<Outline>black</Outline>
+		<Foreground>black</Foreground>
+		<Background>#999999</Background>
+	</PopupStyle>
+
+	<!-- Path where icons can be found.
+	IconPath can be listed multiple times to allow searching
+	for icons in multiple paths.
+	-->
+	<IconPath>/usr/share/pixmaps</IconPath>
+
+	<!-- Virtual Desktops -->
+	<!-- Desktop tags can be contained within Desktops for desktop names. -->
+	<Desktops width="4" height="1">
+		<!-- Default background. Note that a Background tag can be
+		contained within a Desktop tag to give a specific background
+		for that desktop.
+		-->
+		<Background type="gradient">#000000:#737392</Background>
+	</Desktops>
+
+	<!-- Double click speed (in milliseconds) -->
+	<DoubleClickSpeed>400</DoubleClickSpeed>
+
+	<!-- Double click delta (in pixels) -->
+	<DoubleClickDelta>2</DoubleClickDelta>
+
+	<!-- The focus model (sloppy or click) -->
+	<FocusModel>click</FocusModel>
+
+	<!-- The snap mode (none, screen, or border) -->
+	<SnapMode distance="10">screen</SnapMode>
+
+	<!-- The move mode (outline or opaque) -->
+	<MoveMode coordinates="window">outline</MoveMode>
+
+	<!-- The resize mode (outline or opaque) -->
+	<ResizeMode coordinates="window">outline</ResizeMode>
+
+	<!-- Startup commands -->
+	<StartupCommand>xmodmap /etc/Xmodmap.conf</StartupCommand>
+	<StartupCommand>conky</StartupCommand>
+
+	<!-- Key bindings -->
+	<Key key="Up">up</Key>
+	<Key key="Down">down</Key>
+	<Key key="Right">right</Key>
+	<Key key="Left">left</Key>
+	<Key key="h">left</Key>
+	<Key key="j">down</Key>
+	<Key key="k">up</Key>
+	<Key key="l">right</Key>
+	<Key key="Return">select</Key>
+	<Key key="Escape">escape</Key>
+
+	<Key mask="A" key="Tab">nextstacked</Key>
+	<Key mask="A" key="F4">close</Key>
+	<Key mask="A" key="#">desktop#</Key>
+	<Key mask="A" key="F1">root:3</Key>
+	<Key mask="A" key="F2">exec:grun</Key>
+	<Key mask="A" key="F10">maximize</Key>
+	<Key mask="A" key="Right">rdesktop</Key>
+	<Key mask="A" key="Left">ldesktop</Key>
+	<Key mask="A" key="Up">udesktop</Key>
+	<Key mask="A" key="Down">ddesktop</Key>
+	<Key mask="A" key="space">window</Key>
+
+	<Key mask="4" key="Up">maximize</Key>
+	<Key mask="4" key="Down">minimize</Key>
+	<Key mask="4" key="d">showdesktop</Key>
+	<Key mask="4" key="r">exec:grun</Key>
+	<Key mask="4" key="t">exec:aterm</Key>
+	<Key mask="4" key="h">exec:exec:rox</Key>
+	<Key mask="A" key="Return">exec:aterm</Key>
 </JWM>
