diff -rup libXft-2.3.1-orig/xft.pc.in libXft-2.3.1/xft.pc.in
--- libXft-2.3.1-orig/xft.pc.in	2013-11-30 10:05:56.467225683 +0000
+++ libXft-2.3.1/xft.pc.in	2013-11-30 10:06:05.847225934 +0000
@@ -6,7 +6,7 @@ includedir=@includedir@
 Name: Xft
 Description: X FreeType library
 Version: @VERSION@
-Requires: xproto
+Requires:
 Requires.private: xrender, fontconfig, freetype2
 Cflags: -I${includedir}
 Libs: -L${libdir} -lXft
