diff -rup linux-3.14-orig/mm/vmscan.c linux-3.14/mm/vmscan.c
--- linux-3.14-orig/mm/vmscan.c	2014-07-04 16:50:21.578724205 +0300
+++ linux-3.14/mm/vmscan.c	2014-07-04 16:51:54.823722535 +0300
@@ -129,7 +129,7 @@ struct scan_control {
 /*
  * From 0 .. 100.  Higher means more swappy.
  */
-int vm_swappiness = 60;
+int vm_swappiness = 20;
 unsigned long vm_total_pages;	/* The total number of pages which the VM controls */
 
 static LIST_HEAD(shrinker_list);
