diff -rup linux-3.10.29-orig/kernel/printk.c linux-3.10.29/kernel/printk.c
--- linux-3.10.29-orig/kernel/printk.c	2014-02-13 18:08:48.901169169 +0200
+++ linux-3.10.29/kernel/printk.c	2014-02-13 18:09:25.949170160 +0200
@@ -56,7 +56,7 @@
 
 /* We show everything that is MORE important than this.. */
 #define MINIMUM_CONSOLE_LOGLEVEL 1 /* Minimum loglevel we let people use */
-#define DEFAULT_CONSOLE_LOGLEVEL 7 /* anything MORE serious than KERN_DEBUG */
+#define DEFAULT_CONSOLE_LOGLEVEL 3 /* anything MORE serious than KERN_ERR */
 
 int console_printk[4] = {
 	DEFAULT_CONSOLE_LOGLEVEL,	/* console_loglevel */
