diff -rup linux-3.14-orig/init/main.c linux-3.14/init/main.c
--- linux-3.14-orig/init/main.c	2014-07-04 16:50:23.427724172 +0300
+++ linux-3.14/init/main.c	2014-07-04 16:58:09.385715827 +0300
@@ -159,7 +159,7 @@ static int __init set_reset_devices(char
 __setup("reset_devices", set_reset_devices);
 
 static const char * argv_init[MAX_INIT_ARGS+2] = { "init", NULL, };
-const char * envp_init[MAX_INIT_ENVS+2] = { "HOME=/", "TERM=linux", NULL, };
+const char * envp_init[MAX_INIT_ENVS+2] = { "TERM=linux", NULL, };
 static const char *panic_later, *panic_param;
 
 extern const struct obs_kernel_param __setup_start[], __setup_end[];
@@ -868,10 +868,7 @@ static int __ref kernel_init(void *unuse
 		pr_err("Failed to execute %s (error %d).  Attempting defaults...\n",
 			execute_command, ret);
 	}
-	if (!try_to_run_init_process("/sbin/init") ||
-	    !try_to_run_init_process("/etc/init") ||
-	    !try_to_run_init_process("/bin/init") ||
-	    !try_to_run_init_process("/bin/sh"))
+	if (!try_to_run_init_process("/bin/init"))
 		return 0;
 
 	panic("No working init found.  Try passing init= option to kernel. "
@@ -921,7 +918,7 @@ static noinline void __init kernel_init_
 	 */
 
 	if (!ramdisk_execute_command)
-		ramdisk_execute_command = "/init";
+		ramdisk_execute_command = "/bin/init";
 
 	if (sys_access((const char __user *) ramdisk_execute_command, 0) != 0) {
 		ramdisk_execute_command = NULL;
diff -rup linux-3.14-orig/kernel/kmod.c linux-3.14/kernel/kmod.c
--- linux-3.14-orig/kernel/kmod.c	2014-07-04 16:49:38.495724976 +0300
+++ linux-3.14/kernel/kmod.c	2014-07-04 16:57:07.340716938 +0300
@@ -67,11 +67,11 @@ static DECLARE_RWSEM(umhelper_sem);
 /*
 	modprobe_path is set via /proc/sys.
 */
-char modprobe_path[KMOD_PATH_LEN] = "/sbin/modprobe";
+char modprobe_path[KMOD_PATH_LEN] = "/bin/modprobe";
 
 static void free_modprobe_argv(struct subprocess_info *info)
 {
-	kfree(info->argv[3]); /* check call_modprobe() */
+	kfree(info->argv[1]); /* check call_modprobe() */
 	kfree(info->argv);
 }
 
@@ -79,13 +79,11 @@ static int call_modprobe(char *module_na
 {
 	struct subprocess_info *info;
 	static char *envp[] = {
-		"HOME=/",
-		"TERM=linux",
-		"PATH=/sbin:/usr/sbin:/bin:/usr/bin",
+		"PATH=/bin",
 		NULL
 	};
 
-	char **argv = kmalloc(sizeof(char *[5]), GFP_KERNEL);
+	char **argv = kmalloc(sizeof(char *[4]), GFP_KERNEL);
 	if (!argv)
 		goto out;
 
@@ -94,10 +92,8 @@ static int call_modprobe(char *module_na
 		goto free_argv;
 
 	argv[0] = modprobe_path;
-	argv[1] = "-q";
-	argv[2] = "--";
-	argv[3] = module_name;	/* check free_modprobe_argv() */
-	argv[4] = NULL;
+	argv[1] = module_name;	/* check free_modprobe_argv() */
+	argv[2] = NULL;
 
 	info = call_usermodehelper_setup(modprobe_path, argv, envp, GFP_KERNEL,
 					 NULL, free_modprobe_argv, NULL);
