diff -rup /tmp/linux-3.12.23/kernel/printk/printk.c linux-3.12.23/kernel/printk/printk.c
--- /tmp/linux-3.12.23/kernel/printk/printk.c	2014-06-28 12:36:42.901929712 +0300
+++ linux-3.12.23/kernel/printk/printk.c	2014-06-28 12:41:42.418924348 +0300
@@ -59,7 +59,7 @@
 
 /* We show everything that is MORE important than this.. */
 #define MINIMUM_CONSOLE_LOGLEVEL 1 /* Minimum loglevel we let people use */
-#define DEFAULT_CONSOLE_LOGLEVEL 7 /* anything MORE serious than KERN_DEBUG */
+#define DEFAULT_CONSOLE_LOGLEVEL 3 /* anything MORE serious than KERN_ERR */
 
 int console_printk[4] = {
 	DEFAULT_CONSOLE_LOGLEVEL,	/* console_loglevel */
