diff -rup luufs-orig/config.h luufs/config.h
--- luufs-orig/config.h	2014-01-24 22:51:49.227645160 +0200
+++ luufs/config.h	2014-01-24 22:52:01.499644726 +0200
@@ -2,9 +2,9 @@
 #	define _CONFIG_H_INCLUDED
 
 /* the writable layer */
-#define CONFIG_WRITABLE_BRANCH "/tmp"
+#define CONFIG_WRITABLE_BRANCH "/mnt/rw"
 
 /* the read-only layer */
-#define CONFIG_READ_ONLY_BRANCH "/etc"
+#define CONFIG_READ_ONLY_BRANCH "/mnt/ro"
 
 #endif
\ No newline at end of file
