diff -rup mdp-orig/src/Makefile mdp/src/Makefile
--- mdp-orig/src/Makefile	2014-11-20 23:00:54.416797648 +0200
+++ mdp/src/Makefile	2014-11-20 22:59:30.669799148 +0200
@@ -36,7 +36,7 @@ ifeq ($(OS),Windows_NT)
 	endif
 endif
 
-ifeq ($(UNAME_S),Linux)
+ifeq (0,1)
 	LSB_RELEASE := $(shell lsb_release -si 2>/dev/null || echo not)
 	ifneq ($(filter $(LSB_RELEASE),Debian Ubuntu LinuxMint),)
 		CPPFLAGS += -I/usr/include/ncursesw
