diff -rup meh-0.3-orig/src/gif.c meh-0.3/src/gif.c
--- meh-0.3-orig/src/gif.c	2013-12-05 21:09:08.652137111 +0000
+++ meh-0.3/src/gif.c	2013-12-05 21:09:51.052138250 +0000
@@ -57,7 +57,7 @@ static int gif_read(struct image *img){
 	SavedImage *s;
 
 	if(DGifSlurp(g->gif) == GIF_ERROR){
-		PrintGifError();
+		GifErrorString();
 		return 1;
 	}
 
@@ -68,7 +68,7 @@ static int gif_read(struct image *img){
 	else if(g->gif->SColorMap)
 		colormap = g->gif->SColorMap->Colors;
 	else{
-		PrintGifError();
+		GifErrorString();
 		return 1;
 	}
 
