diff -rup minised-1.14-orig/Makefile minised-1.14/Makefile
--- minised-1.14-orig/Makefile	2014-04-16 16:10:36.442891451 +0300
+++ minised-1.14/Makefile	2014-04-16 16:11:03.639890964 +0300
@@ -2,11 +2,11 @@
 
 # If your compiler does not support this flags, just remove them.
 # They only ensure that no new warning regressions make it into the source.
-CFLAGS += -O1 -Wall -Wwrite-strings
+CFLAGS += -Wall -Wwrite-strings
 
-DESTDIR=
+DESTDIR?=
 PREFIX=/usr
-BINDIR=$(PREFIX)/bin
+BINDIR?=$(PREFIX)/bin
 MANDIR=$(PREFIX)/share/man/man1
 
 minised: sedcomp.o sedexec.o
