diff -rup mtools-4.0.18-orig/configure.in mtools-4.0.18/configure.in
--- mtools-4.0.18-orig/configure.in	2014-06-27 12:40:02.745937147 +0300
+++ mtools-4.0.18/configure.in	2014-06-27 12:40:45.351936384 +0300
@@ -96,7 +96,7 @@ AC_CHECK_HEADERS(getopt.h sys/stat.h std
 libc.h fcntl.h limits.h sys/file.h sys/ioctl.h sys/time.h strings.h string.h \
 sys/param.h memory.h malloc.h signal.h sys/signal.h utime.h sgtty.h \
 sys/floppy.h mntent.h sys/sysmacros.h netinet/in.h assert.h \
-iconv.h wctype.h wchar.h locale.h linux/fs.h)
+wctype.h wchar.h locale.h linux/fs.h)
 AC_CHECK_HEADERS(termio.h sys/termio.h, [break])
 AC_CHECK_HEADERS(termios.h sys/termios.h, [break])
 
