diff -rup mtPaint-3368929b5c44fd62e04219c6d2a5054b6b5e678f-orig/src/spawn.c mtPaint-3368929b5c44fd62e04219c6d2a5054b6b5e678f/src/spawn.c
--- mtPaint-3368929b5c44fd62e04219c6d2a5054b6b5e678f-orig/src/spawn.c	2014-10-08 19:08:09.419967722 +0300
+++ mtPaint-3368929b5c44fd62e04219c6d2a5054b6b5e678f/src/spawn.c	2014-10-08 19:10:13.286965504 +0300
@@ -408,28 +408,9 @@ void init_factions()
 #ifndef WIN32
 	int i, j;
 	static char *row_def[][3] = {
-		{"View EXIF data (leafpad)", "exif %f | leafpad"},
-		{"View filesystem data (xterm)", "xterm -hold -e ls -l %f"},
-		{"Edit in Gimp", "gimp %f"},
-		{"View in GQview", "gqview %f"},
-		{"Print image", "kprinter %f"},
-		{"Email image", "seamonkey -compose attachment=file://%f"},
-		{"Send image to Firefox", "firefox %f"},
-		{"Send image to OpenOffice", "soffice %f"},
-		{"Edit Clipboards", "mtpaint ~/.clip*"},
+		{"View in GTKSee", "gtksee %f"},
+		{"Send image to Dillo", "dillo %f"},
 		{"Time delayed screenshot", "sleep 10; mtpaint -s &"},
-		{"View image information", "xterm -hold -sb -rightbar -geometry 100x100 -e identify -verbose %f"},
-		{"#Create temp directory", "mkdir ~/images"},
-		{"#Remove temp directory", "rm -rf ~/images"},
-		{"#GIF to PNG conversion (in situ)", "mogrify -format png *.gif"},
-		{"#ICO to PNG conversion (temp directory)", "ls --file-type *.ico | xargs -I FILE convert FILE ~/images/FILE.png"},
-		{"Convert image to ICO file", "mogrify -format ico %f"},
-		{"Create thumbnails in temp directory", "ls --file-type * | xargs -I FILE convert FILE -thumbnail 120x120 -sharpen 1 -quality 95 ~/images/th_FILE.jpg"},
-		{"Create thumbnails (in situ)", "ls --file-type * | xargs -I FILE convert FILE -thumbnail 120x120 -sharpen 1 -quality 95 th_FILE.jpg"},
-		{"Peruse temp images", "mtpaint ~/images/*"},
-		{"Rename *.jpeg to *.jpg", "rename .jpeg .jpg *.jpeg"},
-		{"Remove spaces from filenames", "for file in *\" \"*; do mv \"$file\" `echo $file | sed -e 's/ /_/g'`; done"},
-		{"Remove extra .jpg. from filename", "rename .jpg. . *.jpg.jpg"},
 //		{"", ""},
 		{NULL, NULL, NULL}
 		},
