diff -rup ratpoison-1.4.6-orig/src/actions.c ratpoison-1.4.6/src/actions.c
--- ratpoison-1.4.6-orig/src/actions.c	2013-11-23 16:04:38.421168950 +0000
+++ ratpoison-1.4.6/src/actions.c	2013-11-23 16:05:51.207441299 +0000
@@ -783,6 +783,7 @@ initialize_default_keybindings (void)
   add_keybinding (XK_Right, 0, "focusright", map);
   add_keybinding (XK_Up, 0, "focusup", map);
   add_keybinding (XK_Down, 0, "focusdown", map);
+  add_keybinding (XK_q, 0, "quit", map);
   add_keybinding (XK_Q, 0, "only", map);
   add_keybinding (XK_R, 0, "remove", map);
   add_keybinding (XK_f, 0, "fselect", map);
