diff -rup ratpoison-1.4.6-orig/src/conf.h ratpoison-1.4.6/src/conf.h
--- ratpoison-1.4.6-orig/src/conf.h	2013-11-29 16:28:40.816679025 +0000
+++ ratpoison-1.4.6/src/conf.h	2013-11-29 16:29:46.360680778 +0000
@@ -120,10 +120,10 @@
 #define TOP_KEYMAP "top"

 /* The default font */
-#define DEFAULT_FONT "-*-fixed-bold-r-normal-*-15-*-*-*-c-*-*-*"
+#define DEFAULT_FONT "*-terminus-medium-*-14-*"
 #define BACKUP_FONT "*"

-#define DEFAULT_XFT_FONT "Mono-11"
+#define DEFAULT_XFT_FONT "Terminus-14"

 /* maximum xvsprintf result string length for systems with pre-C99 snprintf:
  * on errors that are either permanent or cannot be distinguished from those
