diff -rup rox-1.2.2-orig/ROX-Filer/src/pixmaps.c rox-1.2.2/ROX-Filer/src/pixmaps.c
--- rox-1.2.2-orig/ROX-Filer/src/pixmaps.c	2014-01-10 11:57:30.529721651 +0000
+++ rox-1.2.2/ROX-Filer/src/pixmaps.c	2014-01-10 11:59:02.845724125 +0000
@@ -514,7 +514,7 @@ static GdkPixbuf *png_load(png_structp p
 
 	/* Transform grayscale images of less than 8 to 8 bits */
 	if (color_type == PNG_COLOR_TYPE_GRAY && bit_depth < 8)
-		png_set_gray_1_2_4_to_8(png_ptr);
+		png_set_expand_gray_1_2_4_to_8(png_ptr);
 
 	/* Add alpha channel if there's transparency info in the file */
 	if (png_get_valid(png_ptr, info_ptr, PNG_INFO_tRNS))
