diff -rup rox-master-orig/ROX-Filer/src/pinboard.c rox-master/ROX-Filer/src/pinboard.c
--- rox-master-orig/ROX-Filer/src/pinboard.c	2014-06-23 20:50:44.532988941 +0300
+++ rox-master/ROX-Filer/src/pinboard.c	2014-06-23 20:53:40.305985793 +0300
@@ -48,6 +48,7 @@
 #include "bind.h"
 #include "icon.h"
 #include "run.h"
+#include "usericons.h"
 
 /* The number of pixels between the bottom of the image and the top
  * of the text.
@@ -201,6 +202,7 @@ void pinboard_activate(guchar *name)
 {
 	Pinboard	*old_board = current_pinboard;
 	guchar		*path, *slash;
+	int		x, y;
 
 	/* Treat an empty name the same as NULL */
 	if (name && !*name)
@@ -268,10 +270,38 @@ void pinboard_activate(guchar *name)
 		}
 		g_free(path);
 	}
-	else
-		pinboard_pin(home_dir, "Home",
-				4 + ICON_WIDTH / 2,
-				4 + ICON_HEIGHT / 2);
+	else {
+		x = 4 + ICON_WIDTH / 2;
+		y = 4 + ICON_HEIGHT / 2;
+
+		pinboard_pin("/bin/rox", "Home", x, y);
+		set_icon_path("/bin/rox", "/usr/share/pixmaps/home.xpm");
+		pinboard_pin("/bin/settings", "Settings", 64 + x, y);
+		set_icon_path("/bin/settings", "/usr/share/pixmaps/settings.xpm");
+		pinboard_pin("/bin/help", "Help", 128 + x, y);
+		set_icon_path("/bin/help", "/usr/share/pixmaps/help.png");
+		pinboard_pin("/bin/aterm", "Terminal", 196 + x, y);
+		set_icon_path("/bin/aterm", "/usr/share/pixmaps/terminal.xpm");
+
+		pinboard_pin("/bin/dillo", "Dillo", x, 90 + y);
+		set_icon_path("/bin/dillo", "/usr/share/pixmaps/dillo.png");
+		pinboard_pin("/bin/xchat", "X-Chat", 64 + x, 90 + y);
+		set_icon_path("/bin/xchat", "/usr/share/pixmaps/xchat.png");
+		pinboard_pin("/bin/sylpheed", "Sylpheed", 128 + x, 90 + y);
+		set_icon_path("/bin/sylpheed", "/usr/share/pixmaps/sylpheed.png");
+		pinboard_pin("/bin/mtpaint", "mtPaint", 196 + x, 90 + y);
+		set_icon_path("/bin/mtpaint", "/usr/share/pixmaps/mtpaint.png");
+
+		pinboard_pin("/bin/beaver", "Beaver", x, 180 + y);
+		set_icon_path("/bin/beaver", "/usr/share/pixmaps/beaver.png");
+		pinboard_pin("/bin/Ted", "Ted", 64 + x, 180 + y);
+		set_icon_path("/bin/Ted", "/usr/share/pixmaps/Ted.png");
+		pinboard_pin("/bin/siag", "Siag", 128 + x, 180 + y);
+		set_icon_path("/bin/siag", "/usr/share/pixmaps/spreadsheet.xpm");
+		pinboard_pin("/bin/xhippo", "xhippo", 196 + x, 180 + y);
+		set_icon_path("/bin/xhippo", "/usr/share/pixmaps/audio.xpm");
+	}
+
 	loading_pinboard--;
 }
 
