diff -rup squashfs4.2-orig/squashfs-tools/Makefile squashfs4.2/squashfs-tools/Makefile
--- squashfs4.2-orig/squashfs-tools/Makefile	2014-04-21 18:41:21.416964856 +0300
+++ squashfs4.2/squashfs-tools/Makefile	2014-04-21 18:45:16.161960653 +0300
@@ -26,7 +26,7 @@ GZIP_SUPPORT = 1
 # To build using XZ Utils liblzma - install the library and uncomment
 # the XZ_SUPPORT line below.
 #
-#XZ_SUPPORT = 1
+XZ_SUPPORT = 1
 
 
 ############ Building LZO support ##############
