diff -rup ted-master-orig/Ted/tedConfig.h.in ted-master/Ted/tedConfig.h.in
--- ted-master-orig/Ted/tedConfig.h.in	2014-06-10 18:49:04.894993178 +0300
+++ ted-master/Ted/tedConfig.h.in	2014-06-10 18:58:44.585982797 +0300
@@ -56,7 +56,7 @@
 #   endif
 
 #   ifndef	DOCUMENT_DIR
-#	define	DOCUMENT_DIR	PKGDIR "/Ted/"
+#	define	DOCUMENT_DIR	"/usr/share/ted/Ted/"
 #   endif
 
 /************************************************************************/
diff -rup ted-master-orig/appFrame/appFrameConfig.h.in ted-master/appFrame/appFrameConfig.h.in
--- ted-master-orig/appFrame/appFrameConfig.h.in	2014-06-10 18:49:04.930993178 +0300
+++ ted-master/appFrame/appFrameConfig.h.in	2014-06-10 18:58:11.932983382 +0300
@@ -40,11 +40,11 @@
 #   endif
 
 #   ifndef	AFMDIR
-#	define	AFMDIR	PKGDIR "/afm"
+#	define	AFMDIR	"/usr/share/ted/afm"
 #   endif
 
 #   ifndef 	INDDIR
-#	define	INDDIR	PKGDIR "/ind"
+#	define	INDDIR	"/usr/share/ted/ind"
 #   endif
 
 /************************************************************************/
