diff -rup vifm-0.7.8-orig/src/Makefile.in vifm-0.7.8/src/Makefile.in
--- vifm-0.7.8-orig/src/Makefile.in	2014-10-25 18:09:35.000000000 +0300
+++ vifm-0.7.8/src/Makefile.in	2014-11-06 22:22:24.513889480 +0200
@@ -53,7 +53,7 @@ PRE_UNINSTALL = :
 POST_UNINSTALL = :
 build_triplet = @build@
 host_triplet = @host@
-bin_PROGRAMS = vifm$(EXEEXT) vifmrc-converter$(EXEEXT)
+bin_PROGRAMS = vifm$(EXEEXT)
 subdir = src
 DIST_COMMON = $(srcdir)/Makefile.am $(srcdir)/Makefile.in
 ACLOCAL_M4 = $(top_srcdir)/aclocal.m4
@@ -310,8 +310,7 @@ pkgdata_DATA = ../data/vifmrc \
 
 bin_SCRIPTS = vifm-pause
 man_MANS = ../data/vifm.1 \
-					 ../data/vifm-pause.1 \
-					 ../data/vifmrc-converter.1
+					 ../data/vifm-pause.1
 
 MOSTLYCLEANFILES = compile_info.c \
 									 ../data/vifm-help.txt \
