diff -rup wpa_supplicant-2.1-orig/src/drivers/driver_nl80211.c wpa_supplicant-2.1/src/drivers/driver_nl80211.c
--- wpa_supplicant-2.1-orig/src/drivers/driver_nl80211.c	2014-02-12 23:14:16.584448584 +0200
+++ wpa_supplicant-2.1/src/drivers/driver_nl80211.c	2014-02-12 23:25:12.648466152 +0200
@@ -59,6 +59,12 @@
 #define PACKET_TX_TIMESTAMP	16
 #endif
 
+#ifdef CONFIG_LIBNL_TINY
+#define nl_handle nl_sock
+#define nl_handle_alloc_cb nl_socket_alloc_cb
+#define nl_handle_destroy nl_socket_free
+#endif
+
 #ifdef ANDROID
 #include "android_drv.h"
 #endif /* ANDROID */
diff -rup wpa_supplicant-2.1-orig/src/drivers/drivers.mak wpa_supplicant-2.1/src/drivers/drivers.mak
--- wpa_supplicant-2.1-orig/src/drivers/drivers.mak	2014-02-12 23:14:16.584448584 +0200
+++ wpa_supplicant-2.1/src/drivers/drivers.mak	2014-02-12 23:21:12.228459714 +0200
@@ -34,6 +34,7 @@ ifdef CONFIG_LIBNL32
 else
   ifdef CONFIG_LIBNL_TINY
     DRV_LIBS += -lnl-tiny
+    DRV_CFLAGS += -DCONFIG_LIBNL_TINY
   else
     DRV_LIBS += -lnl
   endif
