diff -rup xchat-1.8.9-orig/src/fe-gtk/serverlist.h xchat-1.8.9/src/fe-gtk/serverlist.h
--- xchat-1.8.9-orig/src/fe-gtk/serverlist.h	2014-04-01 23:21:59.167849676 +0300
+++ xchat-1.8.9/src/fe-gtk/serverlist.h	2014-04-01 23:27:04.316844211 +0300
@@ -27,6 +27,10 @@ struct defaultserv
 
 static const struct defaultserv dserv[] =
 {
+	{"", "SUB", "Freenode", 1},
+	{"", "irc.freenode.net", "Freenode", 6667},
+	{"", "ENDSUB", "", 0},
+
 	{"", "SUB", "ChatJunkiesNet", 1},
 	{"#linux", "irc.xchat.org", "ChatJunkiesNet Random Server #linux", 6667},
 	{"#linux", "us.xchat.org", "ChatJunkiesNet U.S. Server #linux", 6667},
