diff -rup xchat-master-orig/src/common/cfgfiles.c xchat-master/src/common/cfgfiles.c
--- xchat-master-orig/src/common/cfgfiles.c	2014-07-04 20:59:07.102884079 +0300
+++ xchat-master/src/common/cfgfiles.c	2014-07-04 21:02:43.467880204 +0300
@@ -37,7 +37,7 @@
 #include <locale.h>
 #endif
 
-#define DEF_FONT "-b&h-lucidatypewriter-medium-r-normal-*-*-120-*-*-m-*-*-*"
+#define DEF_FONT "-b&h-lucidatypewriter-medium-r-normal-*-12-*-*-*-m-*-*-*"
 #define DEF_FONT_JCODE "-*-fixed-medium-r-normal-*-14-*-*-*-c-*-*-*"
 #define DEF_FONT_WIN32 "-*-Fixedsys-normal-r-normal-*-15-*-*-*-m-*-iso8859-1"
 
