diff -rup xmodmap-1.0.8-orig/configure xmodmap-1.0.8/configure
--- xmodmap-1.0.8-orig/configure	2014-04-02 23:20:42.088955390 +0300
+++ xmodmap-1.0.8/configure	2014-04-02 23:24:00.840951830 +0300
@@ -10807,12 +10807,12 @@ if test -n "$XMODMAP_CFLAGS"; then
     pkg_cv_XMODMAP_CFLAGS="$XMODMAP_CFLAGS"
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"x11 xproto >= 7.0.17\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "x11 xproto >= 7.0.17") 2>&5
+    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"x11\""; } >&5
+  ($PKG_CONFIG --exists --print-errors "x11") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_XMODMAP_CFLAGS=`$PKG_CONFIG --cflags "x11 xproto >= 7.0.17" 2>/dev/null`
+  pkg_cv_XMODMAP_CFLAGS=`$PKG_CONFIG --cflags "x11" 2>/dev/null`
 else
   pkg_failed=yes
 fi
@@ -10823,12 +10823,12 @@ if test -n "$XMODMAP_LIBS"; then
     pkg_cv_XMODMAP_LIBS="$XMODMAP_LIBS"
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"x11 xproto >= 7.0.17\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "x11 xproto >= 7.0.17") 2>&5
+    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"x11\""; } >&5
+  ($PKG_CONFIG --exists --print-errors "x11") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_XMODMAP_LIBS=`$PKG_CONFIG --libs "x11 xproto >= 7.0.17" 2>/dev/null`
+  pkg_cv_XMODMAP_LIBS=`$PKG_CONFIG --libs "x11" 2>/dev/null`
 else
   pkg_failed=yes
 fi
@@ -10846,14 +10846,14 @@ else
         _pkg_short_errors_supported=no
 fi
         if test $_pkg_short_errors_supported = yes; then
-	        XMODMAP_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors "x11 xproto >= 7.0.17" 2>&1`
+	        XMODMAP_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors "x11" 2>&1`
         else
-	        XMODMAP_PKG_ERRORS=`$PKG_CONFIG --print-errors "x11 xproto >= 7.0.17" 2>&1`
+	        XMODMAP_PKG_ERRORS=`$PKG_CONFIG --print-errors "x11" 2>&1`
         fi
 	# Put the nasty error message in config.log where it belongs
 	echo "$XMODMAP_PKG_ERRORS" >&5
 
-	as_fn_error $? "Package requirements (x11 xproto >= 7.0.17) were not met:
+	as_fn_error $? "Package requirements (x11) were not met:
 
 $XMODMAP_PKG_ERRORS
 
