diff -rup xsoldier-1.8-orig/Makefile.am xsoldier-1.8/Makefile.am
--- xsoldier-1.8-orig/Makefile.am	2014-06-15 21:45:37.148855675 +0300
+++ xsoldier-1.8/Makefile.am	2014-06-15 21:45:43.836855556 +0300
@@ -68,9 +68,9 @@ AM_CFLAGS = $(X_CFLAGS) $(SDL_CFLAGS) $(
 
 install-exec-local:
 # install pixmap
-	$(mkinstalldirs) $(DESTDIR)$(datadir)/games/xsoldier/
+	$(mkinstalldirs) $(DESTDIR)$(datadir)/xsoldier/
 	$(INSTALL_DATA) pixmap/*.xpm \
-  $(DESTDIR)$(datadir)/games/xsoldier/
+  $(DESTDIR)$(datadir)/xsoldier/
 # install score file
 	$(mkinstalldirs) $(DESTDIR)$(localstatedir)/games/xsoldier/
 	test -e $(DESTDIR)$(localstatedir)/games/xsoldier/xsoldier.scores \
@@ -79,8 +79,8 @@ install-exec-local:
 
 uninstall-local:
 # uninstall pixmap
-	-rm -f $(DESTDIR)$(datadir)/games/xsoldier/*.xpm
-	-rmdir $(DESTDIR)$(datadir)/games/xsoldier/
+	-rm -f $(DESTDIR)$(datadir)/xsoldier/*.xpm
+	-rmdir $(DESTDIR)$(datadir)/xsoldier/
 # uninstall score file
 	-rm -f $(DESTDIR)$(localstatedir)/games/xsoldier/xsoldier.scores
 	-rmdir $(DESTDIR)$(localstatedir)/games/xsoldier/
diff -rup xsoldier-1.8-orig/Makefile.in xsoldier-1.8/Makefile.in
--- xsoldier-1.8-orig/Makefile.in	2014-06-15 21:45:37.147855676 +0300
+++ xsoldier-1.8/Makefile.in	2014-06-15 21:45:43.836855556 +0300
@@ -778,9 +778,9 @@ uninstall-man: uninstall-man6
 
 install-exec-local:
 # install pixmap
-	$(mkinstalldirs) $(DESTDIR)$(datadir)/games/xsoldier/
+	$(mkinstalldirs) $(DESTDIR)$(datadir)/xsoldier/
 	$(INSTALL_DATA) pixmap/*.xpm \
-  $(DESTDIR)$(datadir)/games/xsoldier/
+  $(DESTDIR)$(datadir)/xsoldier/
 # install score file
 	$(mkinstalldirs) $(DESTDIR)$(localstatedir)/games/xsoldier/
 	test -e $(DESTDIR)$(localstatedir)/games/xsoldier/xsoldier.scores \
@@ -789,8 +789,8 @@ install-exec-local:
 
 uninstall-local:
 # uninstall pixmap
-	-rm -f $(DESTDIR)$(datadir)/games/xsoldier/*.xpm
-	-rmdir $(DESTDIR)$(datadir)/games/xsoldier/
+	-rm -f $(DESTDIR)$(datadir)/xsoldier/*.xpm
+	-rmdir $(DESTDIR)$(datadir)/xsoldier/
 # uninstall score file
 	-rm -f $(DESTDIR)$(localstatedir)/games/xsoldier/xsoldier.scores
 	-rmdir $(DESTDIR)$(localstatedir)/games/xsoldier/
diff -rup xsoldier-1.8-orig/xsoldier.h xsoldier-1.8/xsoldier.h
--- xsoldier-1.8-orig/xsoldier.h	2014-06-15 21:45:37.144855676 +0300
+++ xsoldier-1.8/xsoldier.h	2014-06-15 21:46:11.752855056 +0300
@@ -33,7 +33,7 @@ typedef unsigned long Pixel;
 #define WAIT 35000
 #endif /* WAIT */
 
-#define PIXMAP DATADIR "/games/xsoldier/"
+#define PIXMAP DATADIR "/xsoldier/"
 #define SCORE LOCALSTATEDIR "/games/xsoldier/"
 #define SCOREFILE "xsoldier.scores"
 
