diff -rup ytree-1.97-orig/Makefile ytree-1.97/Makefile
--- ytree-1.97-orig/Makefile	2014-10-16 20:31:34.985946294 +0300
+++ ytree-1.97/Makefile	2014-10-16 20:35:04.516942542 +0300
@@ -13,13 +13,13 @@ CC          ?= cc
 # ADD_CFLAGS: Add -DVI_KEYS if you want vi-cursor-keys
 #
 
-DESTDIR     = /usr
+DESTDIR     = /
 
-ADD_CFLAGS  = -O # -DVI_KEYS
+ADD_CFLAGS  = # -DVI_KEYS
 
 BINDIR      = $(DESTDIR)/bin
-MANDIR      = $(DESTDIR)/share/man/man1
-MANESDIR    = $(DESTDIR)/share/man/es/man1
+MANDIR      = $(DESTDIR)/usr/share/man/man1
+MANESDIR    = $(DESTDIR)/usr/share/man/es/man1
 
 
 # Uncomment the lines for your system (default is linux)
@@ -31,9 +31,9 @@ MANESDIR    = $(DESTDIR)/share/man/es/ma
 ###########################
 COLOR       = -DCOLOR_SUPPORT 
 #CLOCK	    = -DCLOCK_SUPPORT # Experimental!
-READLINE    = -DREADLINE_SUPPORT
+READLINE    = 
 CFLAGS      += -D_GNU_SOURCE $(COLOR) $(CLOCK) $(READLINE) $(ADD_CFLAGS)
-LDFLAGS     += -lncurses -lreadline
+LDFLAGS     += -lncurses
 
 ###########################
 # LINUX with Wide character support (testet with ncurses 5.4)
@@ -41,7 +41,7 @@ LDFLAGS     += -lncurses -lreadline
 ###########################
 #COLOR       = -DCOLOR_SUPPORT 
 ##CLOCK	    = -DCLOCK_SUPPORT # Experimental!
-#READLINE    = -DREADLINE_SUPPORT
+#READLINE    = 
 #CFLAGS      = -D_GNU_SOURCE -DWITH_UTF8 $(ADD_CFLAGS) $(COLOR) $(CLOCK) $(READLINE)
 #LDFLAGS     = -lncursesw -lreadline
 
@@ -77,7 +77,7 @@ LDFLAGS     += -lncurses -lreadline
 ###########################
 #COLOR       = -DCOLOR_SUPPORT 
 ##CLOCK	     = -DCLOCK_SUPPORT # Experimental!
-#READLINE    = -DREADLINE_SUPPORT
+#READLINE    = 
 #CFLAGS      = $(ADD_CFLAGS) $(COLOR) $(CLOCK)
 #LDFLAGS     = -lcurses -lcompat -ledit
 #MANDIR      = /usr/share/man/man1
@@ -225,11 +225,8 @@ $(MAIN):	$(OBJS)
 		$(CC) $(LFLAGS) -o $@ $(OBJS) $(LDFLAGS)
 
 install:	$(MAIN)
-		install $(MAIN) $(BINDIR)
-		gzip -9c ytree.1 > ytree.1.gz
-		if [ -d $(MANDIR) ]; then install -m 0644 ytree.1.gz  $(MANDIR)/; fi
-		gzip -9c ytree.1.es > ytree.1.es.gz
-		if [ -d $(MANESDIR) ]; then install -m 0644 ytree.1.es.gz $(MANESDIR)/; fi
+		install -D $(MAIN) $(BINDIR)/$(MAIN)
+		install -D -m 0644 ytree.1  $(MANDIR)/ytree.1
 
 clean:
 		rm -f core *.o *~ *.orig *.bak 
